code/game/machinery/alarm.dm

```diff
@@ -633,20 +639,9 @@
 /obj/machinery/alarm/interact(mob/user)
 	if(buildstage!=2)
 		return
-
-	if ( (get_dist(src, user) > 1 ))
-		if (!istype(user, /mob/living/silicon))
-			user << browse(null, "window=AAlarmwires")
-			return
-
-
-		else if (istype(user, /mob/living/silicon) && aidisabled)
-			to_chat(user, "AI control for this Air Alarm interface has been disabled.")
-			user << browse(null, "window=air_alarm")
-			return
-
-	if(wiresexposed && (isMoMMI(user) || !istype(user, /mob/living/silicon)))
+	if(wiresexposed)
 		wires.Interact(user)
+		return
```

Comment:
I have this change in my other PR to fix a bug already, you fool we're going to conflict!